apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -42,30 +42,48 @@ export default class ChangeAction extends BaseRushAction {
+      'version bumps. Note these changes will eventually be published in a changelog.md file in each package.',
+      '',
+      'The possible types of changes are: ',
+      '',
+      'MAJOR - these are breaking changes that are not backwards compatible. ' +
+      'Examples are: renaming a public class, adding/removing a non-optional ' +
+      'parameter from a public API, or renaming an variable or function that ' +
+      'is exported.',
+      '',
+      'MINOR - these are changes that are backwards compatible (but not ' +
+      'forwards compatible). Examples are: adding a new public API or adding an ' +
+      'optional parameter to a public API',
+      '',
+      'PATCH - these are changes that are backwards and forwards compatible. ' +
+      'Examples are: Modifying a private API or fixing a bug in the logic ' +
+      'of how an existing API works.',
+      ''
+    ];
     super({
       actionVerb: 'change',
       summary: 'Records changes made to projects, indicating how the package version number should be bumped ' +
         'for the next publish.',
-      documentation: ['Asks a series of questions and then generates a <branchname>-<timstamp>.json file ' +
-        'in the common folder. The `publish` command will consume these files and perform the proper ' +
-        'version bumps. Note these changes will eventually be published in a changelog.md file in each package.',
-        '',
-        'The possible types of changes are: ',
-        '',
-        'MAJOR - these are breaking changes that are not backwards compatible. ' +
-        'Examples are: renaming a public class, adding/removing a non-optional ' +
-        'parameter from a public API, or renaming an variable or function that ' +
-        'is exported.',
-        '',
-        'MINOR - these are changes that are backwards compatible (but not ' +
-        'forwards compatible). Examples are: adding a new public API or adding an ' +
-        'optional parameter to a public API',
-        '',
-        'PATCH - these are changes that are backwards and forwards compatible. ' +
-        'Examples are: Modifying a private API or fixing a bug in the logic ' +
-        'of how an existing API works.',
-        ''].join(os.EOL)
+      documentation: documentation.join(os.EOL)
     });
+    try {
+      if (this.rushConfiguration.hotfixChangeEnabled) {
+        // tslint:disable-next-line:no-string-literal
+        documentation.push(
+          'HOTFIX (EXPERIMENTAL) - these are changes that are hotfixes targeting a ' +
+          'specific older version of the package. When a hotfix change is added, ' +
+          'other changes will not be able to increment the version number.' +
+          'Enable this feature by setting \'hotfixChangeEnabled\' in your rush.json.',
+          ''
+        );
+        this.options.documentation = documentation.join(os.EOL);
+      }
+    } catch (e) {
```

Comment:
If it fails, let it fail so that any bug can be exposed. People can still turn hotfixChangeEnabled off to work around the bug.